samples/servicecontrol/adapter-rabbitmq-different-topologies/SCTransportAdapter_1/Shipping/Program.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Threading.Tasks;
+using NServiceBus;
+
+class Program
+{
+    static void Main()
+    {
+        AsyncMain().GetAwaiter().GetResult();
+    }
+
+    static async Task AsyncMain()
+    {
+        Console.Title = "Samples.ServiceControl.RabbitMQAdapter.Shipping";
+
+        #region ShippingConfiguration
+
+        var endpointConfiguration = new EndpointConfiguration("Samples.ServiceControl.RabbitMQAdapter.Shipping");
+
+        var transport = endpointConfiguration.UseTransport<RabbitMQTransport>();
+        transport.ConnectionString("host=localhost");
+        transport.UseDirectRoutingTopology();
+
+        endpointConfiguration.SendFailedMessagesTo("adapter_error");
+        endpointConfiguration.AuditProcessedMessagesTo("adapter_audit");
+        endpointConfiguration.HeartbeatPlugin("adapter_Particular.ServiceControl");
```

Comment:
When I added HBs to the first adapter sample the idea was to show that forwarding of all three queues (audit, error, particular.servicecontrol) works